cats/__init__.py

```diff
@@ -255,4 +255,4 @@ def main(arguments=None):
 
 
 if __name__ == "__main__":
-    main()
+    sys.exit(main())
```

Comment:
Are there practical benefits to always returning from `main` instead of directly calling `sys.exit` from there?

Reply:
Makes it easier to test for failure conditions by checking the return code, alternatively could use pytest.raises(SystemExit) to test this.